packages/flask/src/UICarouselView/CarouselView/Pagination.tsx

```diff
@@ -82,5 +82,6 @@ const styles = StyleSheet.create({
         justifyContent: 'space-between',
         alignSelf: 'center',
         alignItems: 'center',
+        height: UIConstant.carousel.circleSize * 2,
```

Comment:
So you're using `height` like some sort of a padding, right?

Reply:
@savelichalex I just have to pass in the height so the points fit even when scaled. Now the browser calculates the height from non-animated points.

Reply:
Sorry, I didn't get it. Could you explain it another way?